src/agent/agent_diagnostics.js

```diff
@@ -68,7 +68,9 @@ function collect_agent_diagnostics(storage_path) {
                 });
         })
         .finally(() => diag_log.end())
-        .return()
+        .then(() => {
+            // do nothing. 
```

Comment:
I think it's important to note that we "return undefined" and not that we do nothing.
It is even better be explicit like `.then(() => undefined)`

Reply:
sure, I will do it for all those entries.